src/components/overlay/index.jsx

```diff
@@ -231,12 +216,17 @@ export default React.createClass({
     };
   },
 
+  getClassName(){
+    return cx({[this.props.className]:true, 'hull-login__modal':true});
```

Comment:
I do like `cx` but in this case `this.props.className + ' hull-login__modal'` seems clearer. Not a big deal :).